src/components/FindWallet/WalletTable/index.tsx

```diff
@@ -330,7 +332,14 @@ const WalletTable = ({
                 onKeyUp={(e) => {
                   if (e.key === "Enter") showMoreInfo(wallet)
                 }}
-                onClick={() => showMoreInfo(wallet)}
+                onClick={(e) => {
+                  if (
+                    (e.target as HTMLTableRowElement).tagName === "A" ||
+                    (e.target as HTMLTableRowElement).tagName === "svg"
```

Comment:
@Dharmik79 Ahh, good catch with the SVG, but careful because this line will now prevent it from opening for any SVG in this entire box, which there are a few and we don't want that.

I'd suggest we switch to a selector approach, since we only want to get the svg's that are inside an anchor link:

```suggestion
                    (e.target as HTMLElement).matches("a, a svg")
```